test/sorting.test.js

```diff
@@ -16,6 +18,37 @@ import '../vaadin-grid-sorter.js';
 import '../vaadin-grid-sort-column.js';
 
 describe('sorting', () => {
+  describe('attached', () => {
+    it('should populate rows when asynchronously sorted if attached hidden with 0 height', (done) => {
+      const grid = window.document.createElement('vaadin-grid');
+      const col = window.document.createElement('vaadin-grid-column');
+      col.setAttribute('path', 'item');
+      grid.appendChild(col);
+
+      grid.size = 1;
+      grid.dataProvider = function (params, callback) {
+        callback([{ item: 'A' }]);
+      };
+
+      grid.style.maxHeight = '0';
```

Comment:
Is this condition required? If I remove it, the test still passes with the fix and fails without the fix.

Reply:
Removed.